src/Connection.php

```diff
@@ -878,7 +880,7 @@ public function getTransactionNestingLevel(): int
      *
      * If the underlying driver does not support identity columns, an exception is thrown.
      *
-     * @return int|string The last insert ID, as an integer or a numeric string.
+     * @return int|numeric-string The last insert ID, as an integer or a numeric string.
```

Comment:
We cannot guarantee that `lastInsertId()` returns a numeric string and I doubt that we would gain anything even if we could.